test/CMakeLists.txt

```diff
@@ -32,6 +32,13 @@ add_executable(soltest ${sources} ${headers}
 )
 target_link_libraries(soltest PRIVATE libsolc yul solidity yulInterpreter evmasm devcore Boost::boost Boost::program_options Boost::unit_test_framework)
 
+# Special compilation flag for Visual Studio (version 2019 at least affected)
+# in order to compile SolidityEndToEndTest.cpp, which is quite huge.
+# We can remove this flag once we've extracted the tests.
+if (MSVC)
```

Comment:
Want to leave a `FIXME` or `TODO` line here so it is easy to find later?

Reply:
I'm with you @axic, it's just that last time someone asked me to put it out. Will be doing so. :)

Reply:
I'm the first one to revert this PR as soon as I could help reducing the compilation units that cause this. :)